fem/libceed/convection.cpp

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2010-2020, Lawrence Livermore National Security, LLC. Produced
+// at the Lawrence Livermore National Laboratory. All Rights reserved. See files
+// LICENSE and NOTICE for details. LLNL-CODE-806117.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability visit https://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the BSD-3 license. We welcome feedback and contributions, see file
+// CONTRIBUTING.md for details.
+
+#include "convection.hpp"
+
+#include "../../config/config.hpp"
+#ifdef MFEM_USE_CEED
+#include "convection_qf.h"
+#endif
+
+namespace mfem
+{
+
+namespace ceed
+{
+
+#ifdef MFEM_USE_CEED
+struct ConvectionInfo
```

Comment:
I would prefer a name that describes the difference with `ConvectionContext`, maybe `ConvectionKernelInfo`.

Reply:
I went for `ConvectionOperatorInfo`.